scripts/dotnet.py

```diff
@@ -632,10 +633,16 @@ def get_commit_date(
                 success = 1
         except URLError:
             retrycount += 1
+            getLogger().warning(f"URL Error trying to get commit date from {url}, Attempt {retrycount}")
+            sleep(60)
```

Comment:
60 seconds seems like a long time to sleep, maybe do an exponential backoff instead

Reply:
Reimplemented into a backoff with times (in seconds) 10, 20, 40, and 80.

Reply:
This provides 2 shortish, 1 medium, and 1 long wait. Waiting the 80 secs is still faster than rerunning if it works.